ui/lib/apps/Statement/utils/useStatementTableController.ts

```diff
@@ -244,8 +244,20 @@ export default function useStatementTableController(
               errorStrategy: ErrorStrategy.Custom,
             }
           )
-        setStatements(res?.data || [])
-        cacheMgr?.set(cacheKey, res?.data || [])
+        const data = res?.data || []
+
+        // the evicted record's digest will be empty string
+        const evictTips = 'Others(Aggregates all the evicted statements)'
+        data.forEach((d) => {
+          if (d.digest !== '') {
+            return
+          }
+          d.digest = evictTips
+          d.digest_text = evictTips
+        })
+
+        setStatements(data)
+        cacheMgr?.set(cacheKey, data)
```

Comment:
Could you provide a screen shot about how it looks like for "Others" in the list page and in the detail page? Also maybe it is better to display this row as:

*(Others)* (gray)

tooltip: All of other dropped SQL statements

Reply:
Yeah, I need to refine it, hold on.

Reply:
Should we show the detail page when user click evicted data?